tests/0081-admin.c

```diff
@@ -5460,6 +5661,11 @@ static void do_test_apis(rd_kafka_type_t cltype) {
                 /* Describe groups */
                 do_test_DescribeConsumerGroups("temp queue", rk, NULL, -1);
                 do_test_DescribeConsumerGroups("main queue", rk, mainq, 1500);
+        } else {
+                do_test_DescribeConsumerGroups_Compatibility("temp queue", rk,
+                                                             NULL, -1);
+                do_test_DescribeConsumerGroups_Compatibility("main queue", rk,
+                                                             mainq, 1500);
```

Comment:
These all test can be tested in all the scenarios starting Broker version 4.0. Including if condition as well.